### DISCUSSION TITLE
Unable to verify contract

### DISCUSSION THREAD

**Discussion by hakufred** [hashlips-lab/nft-erc721-collection#215] [Q&A]
Hello I have been trying to verify my contract after deploying it to BSC. 

I have done CD smart-contract and have run yarn verify 0x00000… - - network (here I have tried truffle and BSC). I keep getting a hardhat error that the network is not recognized. Please help

**Comment by ghost** [comment]
does it make a difference to where your metamask wallet is connected? is it connected to main net?

**Comment by liarco** [comment]
@hakufred please follow the instructions in my videos (at least from episode 3) and make sure you are not skipping any step: https://github.com/hashlips-lab/nft-erc721-collection#youtube-tutorials

**Reply by hakufred** [reply]
> @hakufred please follow the instructions in my videos (at least from episode 3) and make sure you are not skipping any step: https://github.com/hashlips-lab/nft-erc721-collection#youtube-tutorials

Thanks for the reply. I was using version 2.0.0 of the import file but I started everything over from the beginning with the latest file. I also uninstalled and re-installed All of the necessary programs according to your YouTube video. I am getting Error HH108 : cannot connect to the network truffle. Please make sure your node is running. 

I’m not sure what I’m supposed to do here. Please help.
